src/reconcile/rec_row.c

```diff
@@ -927,9 +927,9 @@ __wt_rec_row_leaf(
                     if (hs_cursor == NULL)
                         WT_ERR(__wt_curhs_open(session, NULL, &hs_cursor));
 
-                    /* From WT_TS_NONE to delete all the history store content of the key. */
-                    WT_ERR(__wt_hs_delete_key_from_ts(session, hs_cursor, btree->id, tmpkey,
-                      WT_TS_NONE, false, F_ISSET(r, WT_REC_CHECKPOINT_RUNNING)));
+                    /* From WT_TS_NONE delete all the history store content of the key. */
+                    WT_ERR(__wt_hs_delete_key_from_ts(
+                      session, hs_cursor, btree->id, tmpkey, WT_TS_NONE, false, false));
```

Comment:
This is a notable change in behavior as previously this call would've returned EBUSY if the WT_REC_CHECKPOINT_RUNNING flag was set and we countered OOO timestamps. I think its okay to not return EBUSY in this case as you're behind a globally visible tombstone.

Reply:
This seems reasonable, though I think it deserves a very clear comment describing why it's OK.